.changeset/fresh-tables-clap.md

```diff
@@ -0,0 +1,75 @@
+---
+"@meilisearch/instant-meilisearch": minor
+---
+
+- Change the behavior of the `or` operator parameter on the [`RefinmentList`](https://www.algolia.com/doc/api-reference/widgets/refinement-list/js/) widget.
+
+⚠️ This impacts the facet distribution number
+
+## Drawback
+
+Currently, for each `facet value` selected in a different `facet` a separate request is made to Meilisearch. Plus an additional one containing all the facets filters. For example, if I select `Adventure` on genres and `Multiplayer` on players, one request is made with both facet filters, one with only `genres=Adventure` and one with only `players=Multiplayer`
+
+In the next release of Meilisearch, the `multi-index` route should be released (see [PR](https://github.com/meilisearch/meilisearch/pull/3417)). When it is released, the work-around will be removed and only one request will be done in all cases!
```

Comment:
Can we promise that?

Reply:
@gmourier can we?

Reply:
It's a multi-search request route if we want to be more precise, and yes, I'm fairly confident we will be able to do deliver it; we already have a working prototype.